tests/providers/amazon/aws/operators/test_ecs.py

```diff
@@ -171,7 +166,7 @@ def test_execute_with_failures(self):
             }
         )
 
-    def test_wait_end_tasks(self):
+    def test_wait_end_tasks(self, aws_hook_mock):
```

Comment:
(Why) Is this necessary?

Reply:
After moving the mock from setUp to above the class, all methods need to have this added (even if unused)

Reply:
Okay, but can you please then add the pylint disable rule only to these lines. So that other unused arguments won't be disabled, too.

Reply:
Found a way to fix the issue with setUp, so this arg will be removed :)